CHANGELOG.rst

```diff
@@ -14,6 +14,7 @@ CHANGELOG
 * feature: HyperparameterTuner: Make input channels optional
 * feature: Add support for Chainer 5.0
 * feature: Estimator: add support for MetricDefinitions
+* feature: source_dir accepts a list of directories
```

Comment:
i still haven't heard an explanation for why this is useful. why can't user just stage their files properly in a single source dir before creating an estimator?

Reply:
From someone on the outside looking in, this looks confusing and unneeded. I understand there's probably a (pressing) reason why we want to do this - but it feels like a hack that we're adding to the PythonSDK to overcome a problem in another system.

Reply:
I am also against this change because of the same reason Jonathan mentioned.

Reply:
User could stage the files before calling the estimator.  User can also copy their own code to S3, and not use the estimator.  Or they could build their own docker container, and make boto calls instead of using the SDK at all.  The point of this project is to make things easier.